selfdrive/car/volkswagen/values.py

```diff
@@ -392,18 +392,24 @@ def init_make(self, CP: car.CarParams):
   CAR.CRAFTER_MK2: {
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x8704L906056EK\xf1\x896391',
+      b'\xf1\x8705L906023BC\xf1\x892688',
     ],
     # Only current upstreamed vehicle has a manual transmission
     #(Ecu.transmission, 0x7e1, None): [
     #],
     (Ecu.srs, 0x715, None): [
       b'\xf1\x873Q0959655BG\xf1\x890703\xf1\x82\x0e16120016130012051G1313052900',
+      b'\xf1\x875QF959655AG\xf1\x890753\xf1\x82\x13151400151500111111000502----1311120749',
+      b'\xf1\x875QF959655AS\xf1\x890755\xf1\x82\x1315140015150011111100050200--1311120749',
```

Comment:
@jyoung8607 do you know why there might be multiple FW from a single car?

Reply:
There wouldn't be. The PR owner works for a shop that does EV conversions on Crafter family vehicles. He's had openpilot plugged in to at least two of them, and isn't familiar with the process.

I can't see his original route due to age, but `af1bf21a07172228|2023-06-29--14-45-28` seems legitimate for a 2023 MAN TGE (a badge-engineered Crafter).

I've also seen a route where he doesn't have engine firmware; that one was post-conversion. I'll try to support him later on those, but for the time being they won't work with openpilot.